contrib/network/connection_balance/dlb/source/connection_balancer_impl.h

```diff
@@ -92,6 +92,9 @@ class DlbConnectionBalanceFactory : public Envoy::Network::ConnectionBalanceFact
 
   std::string name() const override { return "envoy.network.connection_balance.dlb"; }
 
+  Envoy::Network::ConnectionBalancerSharedPtr fallback(const bool is_exact_fallback,
```

Comment:
we are passing the bool by value, no need to make it const as the caller won't be able to tell anyways

Reply:
sure.